mappings/net/minecraft/util/math/ChunkSectionPos.mapping

```diff
@@ -73,6 +75,34 @@ CLASS net/minecraft/class_4076 net/minecraft/util/math/ChunkSectionPos
 	METHOD method_22446 stream (Lnet/minecraft/class_1923;I)Ljava/util/stream/Stream;
 		ARG 0 center
 		ARG 1 radius
+	METHOD method_30551 getPackedLocalPosX (S)I
```

Comment:
`unpackLocalX`, so on

Reply:
That would be inconsistent with the other methods, although I also like that more :-|
Also, "getPacked..." should be "packLocalPos", would you also favor that change together with renaming all of these to "unpack..."?

Reply:
Oh and also inconsistent with the functions to pack/unpack the position of chunk positions themselves.

Reply:
Yeah `getPacked` is accessing a packed value. However there is no packed value to access here: the packed value is rather passed by the method caller and the unpacked value is returned. Hence `unpack` than `getPacked`, and other `getPacked` methods need fixing as well If they are unpacking a value than accessing a packed value.

Reply:
No, what I mean the other method is actually doing the packing :-D
But I take that as a yes.

Reply:
On second thought, I looked through the methods again and I'd rather leave them as is. The class has a shitton of methods dealing with various different coordinate systems, and I think being verbose here - while tedious - avoids mistakes.

Reply:
So this method actually does packing? Are you kidding me, packing a short into an integer?

Reply:
No, but the alternative is: `getXFromPackedLocalPos`.
Regarding pack/unpack: It's not consistent with any of the other methods in this class that do packing.
I.e. `asLong`...

Reply:
Okay, I refactored more members in this class and added alot more docs. Is this better?